inc/kmer_unordered_set.hpp

```diff
@@ -22,19 +27,30 @@ class KmerSets {
 		// path to kmer parent files as input from command line
 		path hap1_kmer_fa_path;
 		path hap2_kmer_fa_path;
-
-		int hap1_kmer_count;
-		int hap2_kmer_count;
+		float num_hap1_kmers;
+		float num_hap2_kmers;
+		// component and haplotype 
+		size_t graph_component;
+		size_t component_haplotype;
+		static const size_t parent_hap1_index = 0; // hg03 paternal 
+		static const size_t parent_hap2_index = 1; // hg04 maternal
+		// < component,  [component_hap_path][parent_hap_index] >
+		map<size_t, array <array <float,2>, 2>> component_map;
 
 	/// Methods ///
 	public:
 		KmerSets();
-		KmerSets(path hap1_kmer_fa_path_arg, path hap2_kmer_fa_path_arg);
-		void load_file_into_unordered_set(path file_path, unordered_set <string>& set);
+		KmerSets(path hap1_kmer_fa_path_arg, path hap2_kmer_fa_path_args);
+		float get_size_of_kmer_file(path);
+		void load_file_into_unordered_set(path , unordered_set <string>& );		
 		void get_parent_kmer_sets();
-		bool find_haplotype_kmer_set_count(unordered_set <string>);
-		bool find_haplotype_single_kmer_count(string child_kmer);
-
+		void parse_path_string(string);
+		bool increment_parental_kmer_count(string, string );
+		bool increment_parental_kmer_count(string, unordered_set <string>);
+		void normalize_kmer_counts();
+		void print_component_parent_conf_matrix();
```

Comment:
I just noticed that there are no variable names here. Ideally they would be here so that they can be used as a hint when you are looking up the functions in your IDE later.